simulator.go

```diff
@@ -266,14 +266,26 @@ func (h *simulatorAPIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request)
 				http.Error(w, err.Error(), http.StatusInternalServerError)
 				return
 			}
-			logger = logger.New("id", container.ID[:8])
+			cLogger := logger.New("id", container.ID[:8])
```

Comment:
Why did you rename this? It will cause the loggers below not to have the `id` embedded any more. This will especially be bad for the new termination logs you added, since they don't have any context, so you'll just get a batch of `client container finished cleanly` messages one after the other. Am I understanding the code incorrectly?